### ISSUE TITLE
Which LSTM for predicted probabilities on test?

### ISSUE THREAD

**Issue by hslh** [simondewit/pickleteam#14]
I'm about to run the LSTM to get the predicted probabilities for the test data, but should I use the original kerasNetwork.py or pickleteam_new/neuralNetworkClassifier.py ? And do we want to use dev data as additional training data?

**Comment by DaphneGroot** [comment]
Sorry for the late reply, I've had kind of a weird day.
you can use the pickleteam_new/neuralNetworkClassifier.py, by calling `python3 main.py --method='neural' --predict_language='e'/'s'` (e for English, s for Spanish).

I haven't thought about using the dev data as training data. As of now, that is not implemented, since we still also predict on the dev set to keep track of our scores.

**Comment by hslh** [comment]
No problem, I can just run it today and it should be finished in the late afternoon. I think not using the dev data is fine, I just wondered about that because one of the commits mentioned something like that. Thanks for the info!

**Comment by hslh** [comment]
I added Spanish. The labels were already output by the existing code, I added in (but not committed) something ugly to also pickle the predicted class probabilities. This should be the output you need, but let me know if it needs changing. English is now running.

**Comment by hslh** [comment]
Added English.
